pkg/clusterregistryconfig/flags.go

```diff
@@ -0,0 +1,369 @@
+package clusterregistryconfig
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/openshift/rosa/pkg/helper"
+	"github.com/openshift/rosa/pkg/input"
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/ocm"
+)
+
+const (
+	allowedRegistriesFlag          = "allowed-registries"
```

Comment:
Can we include a prefix of registry-config on these? It helps when looking at the help output or understanding what the flags are for

Reply:
Thanks so much @gdbranco , what kind of prefix are you referring to?

Reply:
Like instead of 'allowed-registries' something like 'registry-config-allowed-registries' just so it is clear for what the flag is for

Example, '--default-ingress-excluded-namespaces' this makes it clear it is a setting for the default ingress

Reply:
Ahhh got it! Will change the name